### PR TITLE
Address operations: done flake8 fixes and rewrited

### PR DESCRIPTION
**Pull request by g1itch**
Hello.

This is my variant of cleanup for code related to address operations. I picked it out from my keystore-plugin branch.

### CODE REVIEW

**Comment by g1itch** [comment]
As for `tr._translate()` which codacy considers as "Access to a protected member _translate of a client class" I can rewrite that code with `from tr import _translate` using `_translate()` instead like it done in ui-related modules.

**Comment by g1itch** [comment]
Also should I replace `privSigningKeyHex` with underscore. What's your current convention about unused variables in tuple assignment?

**Comment by PeterSurda** [comment]
@g1itch forget about `_tr` at the moment. Regarding unused variables, use `_` and then put a comment in the previous (or next) line with all named variables.

File: src/class_singleWorker.py [hunk 1]
```diff
@@ -390,383 +452,623 @@ def sendOutOrStoreMyV4Pubkey(self, myAddress):
...
+                            ackdata,
+                            tr._translate(
+                                "MainWindow",
+                                "Problem: You are trying to send a"
+                                " message to yourself or a chan but your"
+                                " encryption key could not be found in"
+                                " the keys.dat file. Could not encrypt"
+                                " message. %1"
+                            ).arg(l10n.formatTimestamp()))
+                    ))  # log or show the address maybe?
```
Comment (PeterSurda) [inline src/class_singleWorker.py]: Well, this changes the status, so the recipient address is in the same row of the message list.

**Comment by g1itch** [comment]
"Use implicit string literal continuation to break that literal across multiple lines inside the parens" means to not use `\` to break the strings. Any consequent strings are concatenated implicitly, inside parens you don't need `\` for line continuation.

**Comment by g1itch** [comment]
I have a problem with codacy... https://ibb.co/irwZzc

**Comment by g1itch** [comment]
@PeterSurda, should I turn `_getKeysForAddress` and `_doPOWDefaults` into functions or maybe staticmethod?

**Comment by PeterSurda** [comment]
@g1itch maybe separately, I don't want to do big PRs as it's difficult to review. I'll merge this now.

**Pull request merged**